climada/hazard/storm_europe.py

```diff
@@ -234,27 +240,30 @@ def read_cosmoe_file(self, fp_file, run_date, event_date=None,
         ncdf = ncdf.assign_coords(date=('time',ncdf["time"].dt.floor("D")))
 
         if event_date:
-            # stacked = np.max(ncdf.sel(time=event_date.strftime('%Y-%m-%d')).VMAX_10M,
-            #                  axis=0).stack(intensity=('y_1', 'x_1'))
-            stacked2 = ncdf.sel(time=event_date.strftime('%Y-%m-%d')
-                                ).groupby('date'
-                                          ).max().stack(intensity=('y_1',
-                                                                   'x_1'))
+            try:
+                stacked2 = ncdf.sel(time=event_date.strftime('%Y-%m-%d')
+                                    ).groupby('date'
+                                              ).max().stack(intensity=('y_1',
+                                                                       'x_1'))
+            except KeyError:
+                raise ValueError('The selected event_date {} is not contained'
```

Comment:
```suggestion
                raise ValueError('Extraction of date and coordinates failed. '
                                 'This is most likely because '
                                 'the selected event_date {} is not contained'
```
This is really picky, but there are other, less likely, reasons you could get an error here (e.g. MeteoSwiss changes the coordinate names), so I'm suggesting a longer message.

Reply:
Done